Tasks/DotNetCoreCLIV2/Tests/L0.ts

```diff
@@ -312,6 +312,19 @@ describe('DotNetCoreExe Suite', function () {
         done();
     });
 
+
+    it('publish works with publishWebProjects option if .csproj have Microsoft.Net.Sdk.Web', (done: MochaDone) => {
+        process.env["__projects__"] = "havesdk*/*.csproj;";
```

Comment:
how is this test testing your new feature

Reply:
We are mocking inputfile argument and defining a mapping for the argument, then we're mocking the output of that definition as well.